notificationworkerlambda/src/main/scala/com/gu/notifications/worker/AndroidSender.scala

```diff
@@ -70,14 +71,14 @@ class AndroidSender(val config: FcmWorkerConfiguration, val firebaseAppName: Opt
       .through(cloudwatch.sendResults(env.stage, Configuration.platform))
   }
 
-  def reportBatchLatency[C <: DeliveryClient](chunkedTokens: ChunkedTokens, notificationSentTime: Instant): Pipe[IO, Either[DeliveryException, BatchDeliverySuccess], Unit] = { input =>
+  def reportBatchLatency[C <: DeliveryClient](chunkedTokens: ChunkedTokens, metadata: Option[NotificationMetadata]): Pipe[IO, Either[DeliveryException, BatchDeliverySuccess], Unit] = { input =>
     val shouldPushMetricsToAws = chunkedTokens.notification.dryRun match {
       case Some(true) => false
       case _ => true
     }
     input
-      .fold(List.empty[Long]) { case (acc, resp) => LatencyMetrics.collectBatchLatency(acc, resp, notificationSentTime) }
-      .through(cloudwatch.sendLatencyMetrics(shouldPushMetricsToAws, env.stage, Configuration.platform, Reporting.notificationTypeForObservability(chunkedTokens.notification)))
+      .fold(List.empty[Long]) { case (acc, resp) => LatencyMetrics.collectBatchLatency(acc, resp, metadata.map(_.notificationAppReceivedTime).getOrElse(Instant.now())) } // FIXME: remove this fallback after initial deployment
+      .through(cloudwatch.sendLatencyMetrics(shouldPushMetricsToAws, env.stage, Configuration.platform, metadata.flatMap(_.audienceSize)))
```

Comment:
This change does 2 things. 
* It changes the source data format to the new NotificationMetadata case class;
* It replaces the notification-type parameter with one that tracks audience size. 
The reason for this switch to audience size is:
1. There is no functional difference between the handling of Breaking News and other notifications;
2. The size of Breaking News notifications varies wildly between 10,000 and more than 1,000,000 (see the [Ophan notifications page](https://dashboard.ophan.co.uk/notifications) for examples).
The size of these notifications has a very significant impact on the latency of the notification process and it is unrealistic to compare one of the small size notifications with the large notifications. This change allows us to bucket notifications by size instead, comparing like with like.